packages/chrome.rb

```diff
@@ -0,0 +1,41 @@
+require 'package'
+
+class Chrome < Package
+  description 'Google Chrome is a fast, easy to use, and secure web browser.'
+  homepage 'https://www.google.com/chrome'
+  compatibility 'x86_64'
+  @_ver = '87.0.4280.141'
+  @_deb = "google-chrome-stable_#{@_ver}-1_amd64.deb"
+  version @_ver
+
+  case ARCH
+  when 'x86_64'
+    source_url "https://dl.google.com/linux/deb/pool/main/g/google-chrome-stable/#{_deb}"
+    source_sha256 'b7edb7cd5c166bf3c0a1d245baa5924e242c3b81b97090468bec778f41f40373'
+    depends_on 'alien' => :build
+    depends_on 'nspr'
+    depends_on 'cairo'
+    depends_on 'gtk3'
+    depends_on 'expat'
+    depends_on 'cras'
+    depends_on 'sommelier'
+  end
+
+  def self.build
+    Dir.chdir '..' do
+      system "alien -tc #{@_deb}"
+      system "tar xvf google-chrome-stable-#{@_ver}.tgz > /dev/null"
```

Comment:
Maybe it's just me but I always cringe when I see `> /dev/null` because it hides the output.  It doesn't hurt anything to see everything during the build.

Reply:
I think it is fine with 'tar' or 'unzip' as `crew` also hide them

Reply:
The `v` in `xvf` means verbose.

Reply:
Removed